dotCMS/src/main/java/com/dotmarketing/common/db/DotConnect.java

```diff
@@ -1173,4 +1180,17 @@ public int executeUpdate (final Connection connection, final String preparedStat
         return rowsAffected;
     } // executeUpdate.
 
+    private static void setTimestampWithTimezone(PreparedStatement statement, int parameterIndex, Object timestamp) {
+
+        final DotTimezonedTimestamp dotTimezonedTimestamp = (DotTimezonedTimestamp) timestamp;
+        final Calendar calendar = Calendar.getInstance(TimeZone.getTimeZone(dotTimezonedTimestamp.getTimezone()));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use the Java 8 Date and Time API instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2143)